package.json

```
@@ -3,16 +3,17 @@
   "version": "1.1.2",
   "description": "Copy element text content or input values to the clipboard.",
   "repository": "github/clipboard-copy-element",
-  "main": "dist/index.umd.js",
-  "module": "dist/index.esm.js",
-  "types": "index.d.ts",
+  "main": "dist/index.js",
+  "type": "module",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
   "scripts": {
     "clean": "rm -rf dist",
     "lint": "github-lint",
-    "prebuild": "npm run clean && npm run lint",
-    "build": "rollup -c && cp src/clipboard-copy-element.js.flow dist/index.esm.js.flow && cp src/clipboard-copy-element.js.flow dist/index.umd.js.flow",
+    "prebuild": "npm run clean && npm run lint && tsc",
```

Do we need `--noEmit` here?

```suggestion
    "prebuild": "npm run clean && npm run lint && tsc --noEmit",
```

---

`tsc` compiles the code to JS which `rollup` uses here:

https://github.com/github/clipboard-copy-element/blob/89277d9910e0c60cdec83d2ed6cc09e37e25e495/rollup.config.js#L4